packages/core-components/api-report.md

```diff
@@ -510,6 +510,11 @@ export type IconLinkVerticalProps = {
   title?: string;
 };
 
+// Warning: (ae-missing-release-tag) "IdentityProviders" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
```

Comment:
I think the `IdentityProviders` type will need a `/** @public **/` tsdoc notation to remove this warning :)

Reply:
Thanks! I've updated that now!